resources/views/forum/topics.twig

```diff
@@ -1,5 +1,15 @@
 {% extends 'forum.home' %}
 
+{% set current_tag = renderParams.tagName|default(null) %}
+
+{% block title %}
+  {{ current_tag }} | Problemy i Porady :: 4programmers.net
```

Comment:
to pewnie trzeba byłoby poprawić tak, żeby wyświetlało coś sensownego podczas otwierania widoków, do których linki podałem w poprzednim komentarzu

Reply:
Poprawka w #730